### DISCUSSION TITLE
Only show configured tasts?

### DISCUSSION THREAD

**Discussion by Barugon** [zed-industries/zed#15751] [Feature Requests]
Is there any way to have the tasks window (Alt + Shift + T) show only the configured tasks (i.e. no auto-detected tasks, no history)?

I want to be able to show only the tasks that I have configured for a specific project.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
